src/boot/efi/efi-string.c

```diff
@@ -160,9 +160,12 @@ int efi_memcmp(const void *p1, const void *p2, size_t n) {
 }
 
 void *efi_memcpy(void * restrict dest, const void * restrict src, size_t n) {
-        if (!dest || !src || n == 0)
+        if (n == 0)
```

Comment:
I dislike this. The whole point for the strings library is to also be NULL pointer safe and doing something sensible rather than getting our asses kicked at runtime in obscure corner cases. Assert would certainly work, but not with `NDEBUG` builds.

Reply:
Hmm... Then, we should slightly rename it to emphasize that this accepts NULL even if `n != 0`. At least, a brief comment should be added.

Reply:
OK. Will drop the commit.